src/DurableTask.Netherite/Scaling/ScalingMonitor.cs

```diff
@@ -249,6 +247,22 @@ public async Task<string> TaskHubIsIdleAsync(Dictionary<uint, PartitionLoadInfo>
 
             // we have concluded that there are no pending work items, timers, or unprocessed input queue entries
             return null;
-        }  
+        }
+
+        public static async Task<List<long>> GetQueuePositionsAsync(ConnectionInfo connectionInfo, string partitionHub)
+        {
+            var client = connectionInfo.CreateEventHubConsumerClient(partitionHub);
+            try
+            {
+                var partitionIds = await client.GetPartitionIdsAsync();
+                var infoTasks = partitionIds.Select(id => client.GetPartitionPropertiesAsync(id)).ToList();
+                await Task.WhenAll(infoTasks);
+                return infoTasks.Select(t => t.Result.LastEnqueuedSequenceNumber + 1).ToList();
+            }
+            catch (Azure.Messaging.EventHubs.EventHubsException)
```

Comment:
This should catch only the *specific* error where the event hub does not exist. The reason why we catch this specific exception is because it happens normally - for new task hubs the event hub may not have been created yet and in that case we want the scale controller to not report any exceptions.

However, for other exceptions (for example, unsufficient permissions) we *do* want the exception to be thrown and reported by the scale controller.